src/Data/Dictionary/DictionaryItem.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+namespace LORIS\Data\Dictionary;
+
+use \LORIS\Data\Scope;
+use \LORIS\Data\Type;
+use \LORIS\Data\Cardinality;
+
+/**
+ * A DictionaryItem represents a description of a type of data
+ * managed by LORIS.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class DictionaryItem implements \LORIS\StudyEntities\AccessibleResource
+{
+    protected $name;
+    protected $description;
+    protected $scope;
+    protected $typ;
```

Comment:
`typ` ?? not `type` ?

Reply:
`type` is a reserved keyword in php.

Reply:
at least I thought it was and editor was highlighting it as one.. I'll look into it a little more.